symphony/versions.json

```diff
@@ -79,6 +73,30 @@
       },
       "previous_version_name": "v28",
       "next_version_name": ""
+    },
+    {
+      "name": "v1.0.7",
+      "tag": "v1.0.7",
+      "height": 3333145,
+      "recommended_version": "v1.0.7",
+      "compatible_versions": ["v1.0.7"],
+      "binaries": {
+        "linux/amd64": "https://github.com/Orchestra-Labs/symphony/releases/download/v1.0.7/symphonyd"
+      },
+      "consensus": {
+        "type": "cometbft",
+        "version": "v0.38.12"
```

Comment:
Path: consensus.version, Reason: Value mismatch, In "codebase": 0.38.11, In currentVersion: v0.38.12